scripts/tts_dataset_files/ipa_cmudict-0.7b_nv22.10.txt

```diff
@@ -130342,7 +130342,6 @@ WHAT'VE  ˈwʌtəv
 WHAT-WHAT  ˈwʌtˈwʌt
 WHATE  ˈweɪt
 WHATEVER  ˌwʌˈtɛvɚ
-WHATEVER(1)  ˌhwʌˈtɛvɚ
 WHATEVER'S  ˌwʌˈtɛvɚz
 WHATEVER'S(1)  ˌhwʌˈtɛvɚz
```

Comment:
Can you update whatever too?
And move the changes to the bottom of the dictionary

Reply:
updated